config/tutorials/otel-trace/index-config.yaml

```diff
@@ -5,7 +5,7 @@
 
 version: 0.6
 
-index_id: otel-trace-v0
+index_id: otel-traces-v0_6
```

Comment:
can you update the whole index config? I think the mapping is not up to date

Reply:
This config file seems useless, do you know where it's used?

Reply:
if this index config is not used in a tutorial, can you remove it?

Reply:
But then we need to maintain this file. The trace index config is automatically created. You can use the API to get the index config for example. 
What is shown in the docs should the index config written in the rust code.

Currently the directory is `config/tutorials` and if it's not used for tutorials then why keeping it? It's confusing.